packages/lesswrong/components/forumEvents/DonationElectionLeaderboard.tsx

```diff
@@ -0,0 +1,263 @@
+import React, { useMemo, useState } from "react";
+import { Components, registerComponent } from "../../lib/vulcan-lib";
+import { IRPossibleVoteCounts } from "@/lib/givingSeason/instantRunoff";
+import { ACTIVE_DONATION_ELECTION, DONATION_ELECTION_NUM_WINNERS } from "@/lib/givingSeason";
+import { useMulti } from "@/lib/crud/withMulti";
+import classNames from "classnames";
+import { Link } from "@/lib/reactRouterWrapper";
+import { GIVING_SEASON_MD_WIDTH } from "./useGivingSeasonEvents";
+
+const styles = (theme: ThemeType) => ({
+  root: {
+    display: "flex",
+    flexDirection: "column",
+    gap: "16px",
+    flex: 1,
+    maxWidth: 750,
+    marginBottom: 16
+  },
+  header: {
+    fontSize: 18,
+    marginBottom: 26,
+    fontWeight: 600
+  },
+  candidateWrapper: {
+    display: "flex",
+    gap: "12px"
+  },
+  candidateImage: {
+    width: '36px',
+    height: '36px',
+    backgroundSize: 'cover',
+    backgroundPosition: 'center',
+    borderRadius: theme.borderRadius.small
+  },
+  result: {
+    width: "100%"
+  },
+  candidateDetails: {
+    display: "flex",
+    justifyContent: "space-between"
+  },
+  candidateName: {
+    fontSize: 14,
+    fontWeight: 600,
+    textDecoration: "underline",
+    textUnderlineOffset: "3px",
+    maxWidth: 350,
+    whiteSpace: "nowrap",
+    overflow: "hidden",
+    textOverflow: "ellipsis",
+    '&:hover': {
+      textDecoration: "underline",
+      textUnderlineOffset: "3px",
+    }
+  },
+  voteCount: {
+    fontWeight: 500,
+    color: theme.palette.inverseGreyAlpha(0.8)
```

Comment:
I think all these `inverseGreyAlpha`s should actually be
```
color: theme.palette.text.alwaysWhite,
opacity: 0.whatever,
```
otherwise they'll look different in light mode vs dark mode, which probably isn't what we want